templates/base24.mustache

```diff
@@ -0,0 +1,111 @@
+Windows Registry Editor Version 5.00
+
+; Scheme name: {{scheme-name}}
+; Scheme system: {{scheme-system}}
+; Scheme author: {{scheme-author}}
+; Template author: Tinted Theming (https://github.com/tinted-theming)
+[HKEY_CURRENT_USER\Software\SimonTatham\PuTTY\Sessions\{{scheme-system}}-{{scheme-slug}}]
+
+; Default Foreground
+; base05 #{{base05-hex}}
+"Colour0"="{{base05-rgb-r}},{{base05-rgb-g}},{{base05-rgb-b}}"
+
+; Default Bold Foreground  -- equals to non-bold
+; base05 #{{base05-hex}}
+"Colour1"="{{base05-rgb-r}},{{base05-rgb-g}},{{base05-rgb-b}}"
+
+; Default Background
+; base00 #{{base00-hex}}
+"Colour2"="{{base00-rgb-r}},{{base00-rgb-g}},{{base00-rgb-b}}"
+
+; Default Bold Background  -- equals to non-bold
+; base00 #{{base00-hex}}
+"Colour3"="{{base00-rgb-r}},{{base00-rgb-g}},{{base00-rgb-b}}"
+
+; Cursor Text -- equals to default background
+; base00 #{{base00-hex}}
+"Colour4"="{{base00-rgb-r}},{{base00-rgb-g}},{{base00-rgb-b}}"
+
+; Cursor Colour -- equals to default foreground
+; base05 #{{base05-hex}}
+"Colour5"="{{base05-rgb-r}},{{base05-rgb-g}},{{base05-rgb-b}}"
+
+; ANSI Black
+; 30m
+; base00 #{{base00-hex}}
+"Colour6"="{{base00-rgb-r}},{{base00-rgb-g}},{{base00-rgb-b}}"
+
+; ANSI Black Bright
+; 1;30m
+; base03 #{{base02-hex}}
```

Comment:
These comments probably have to be reworked to actually express what is going on...
They seem to be a bit misleading the way they are right now (there is a mismatch between the number in the comment and the variable name)...